hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,10 @@
+RELEASE_TYPE: minor
+
+This release makes it an explicit error to call
+:func:`floats(min_value=inf, exclude_min=True) <hypothesis.strategies.floats>` or
+:func:`floats(max_value=-inf, exclude_max=True) <hypothesis.strategies.floats>`,
+as there are no possible values that can be generated (:issue:`1859`).
+
+:func:`floats(min_value=0.0, max_value=-0.0) <hypothesis.strategies.floats>`
+is now deprecated.  While comparison operators would allow ``0.0 <= x <= -0.0``
+for x as either signed zero, the interval is negative and thus disallowed.
```

Comment:
I don't think "the interval is negative" makes sense as a phrase. What do you intend it to mean?